packages/components/constraints/src/helpers.ts

```diff
@@ -24,7 +25,13 @@ const parseHorizontalConstraintProp = (prop: number | string) => {
     // prop is of type `max`
     return prop;
   }
+
+  // @TODO: remove this dependency
   // prop is of type `constraint`
+  warning(
+    false,
+    'Component is using old constraint values. Please use one of `max` constraint values.'
+  );
```

Comment:
I added this warning, incase a consumer does the following

```
// should use one of `max` values instead.
<DateField horizontalConstraint="xs" /> 
```

Reply:
We should remove any usage/reference of the old constraints and therefore we should not accept those values anymore.

Reply:
I removed it all.
This is a real breaking change now 😄